charts/tailscale-operator/templates/operator-cluster-role.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
The ClusterRole defines the necessary access for watching resources (Services, Ingresses) and could be bound either with a ClusterRole if the operator is watching the entire cluster, or by targeted RoleBindings if it is restricted to specific namespaces.